pykafka/handlers.py

```diff
@@ -86,6 +94,22 @@ def spawn(self, target, *args, **kwargs):
         return t
 
 
+class GEventHandler(Handler):
+    """A handler that uses a greenlet to perform its work"""
+    Queue = JoinableQueue
+    Event = GEEvent
+    Lock = GERLock  # fixme
+    RLock = GERLock
```

Comment:
This is mostly a style thing, but if you're only using it one place, the extra characters to have this be `gevent.event.Event` will make it a lot clearer to read. Otherwise, you're using a work `GEEvent` which nobody will have seen before. Not a big thing, but it'll help readability, IMO.